### PR TITLE
feat: Add PySimAI

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**

### CODE REVIEW

File: doc/source/api.rst [hunk 1]
```diff
@@ -165,6 +165,12 @@ PyAnsys packages API reference
       :text-align: center
       :class-title: pyansys-card-title
 
+    .. grid-item-card:: PySimAI
+      :img-top: _static/thumbnails/pysherlock.png
```
Comment (RobPasMue) [inline doc/source/api.rst:169]:
```suggestion
      :img-top: _static/thumbnails/intro.png

```

File: doc/source/index.rst [hunk 2]
```diff
@@ -210,6 +210,14 @@ it is now a collection of many Python packages for using Ansys products through
 
       Pythonic interface to communicate with Ansys Sherlock
 
+    .. grid-item-card:: PySimAI
+      :img-top: _static/thumbnails/pysherlock.png
```
Comment (RobPasMue) [inline doc/source/index.rst:214]:
```suggestion
      :img-top: _static/thumbnails/intro.png
```

**Review by RobPasMue — APPROVED** [review]

**Comment by tmpbeing** [comment]
Just noticed we were added to PyAnsys. Thank you guys !

**Comment by RobPasMue** [comment]
Anytime @tmpbeing  - if you want a custom image let us know. It has to be 640x480 though. That's the only request =)

**Comment by camzeech** [comment]
Hello all, would this picture work? It's not the best we have but it would be a good start still.

Bracket (no blur)

**Comment by RobPasMue** [comment]
Of course. Let me add it

**Pull request merged**